tests/test_config.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+
+cwd = os.getcwd()
+if cwd not in {'root/project', '/opt/code/tap-square'}:
+    print("WARN: This script is meant to run from the top level directory of the tap.")
```

Comment:
Will these directories work in circle? I assume so but double checking. Also, there's no reason this test shouldn't be able to run from any directory (like ~/git/tap-square). But I guess this is just a print warning and not an assertion... but I'm not sure what value it provides.

Reply:
It does matter where it is ran from. It works in those two directories. I think it will fail anywhere else because it will mess up the path. I'm not great at working with paths, not really sure how to make it work from anywhere.

Reply:
I'm confused why that is the case if you're using cwd as the prefix in the file search... But there is a way in python to expand a path to the full path in case that is necessary. Leave this for now, someone can fix it later when necessary.